### DISCUSSION TITLE
Custom fields for entries similar to KeeWeb

### DISCUSSION THREAD

**Discussion by mackayem** [keepassxreboot/keepassxc#6271] [Ideas]
First of all, thank you for all the contributors hard work. This password manager has made my life much easier in ensuring I keep my online personal online organized, secure and up to date.

I was considering switching to KeeWeb, but prefer a lot of features of KeePassXC more. One feature that I enjoy about KeeWeb is the ability to add custom fields to entries. This allows me to add an Email field and any other important information such as if my address or name is contained within the account, or credit cards used etc.

Curious if any other users would find this feature useful as well?

Cheers!

**Comment by droidmonkey** [comment]
You can add advance attributes from the advanced tab when editing an entry. We are working on custom entry layouts for 2.7.0 to make it more of a first class feature.
